node/rpc.go

```diff
@@ -73,6 +73,10 @@ func FullNodeHandler(a v1api.FullNode, permissioned bool, opts ...jsonrpc.Server
 		rpcServer.Register("Filecoin", hnd)
 		rpcServer.AliasMethod("rpc.discover", "Filecoin.Discover")
 
+		rpcServer.AliasMethod("eth_accounts", "Filecoin.EthAccounts")
+		rpcServer.AliasMethod("eth_blockNumber", "Filecoin.EthBlockNumber")
+		rpcServer.AliasMethod("eth_getBlockTransactionCountByNumber", "Filecoin.EthGetBlockTransactionCountByNumber")
```

Comment:
This aliasing is fine, but could be made a little easier (to maintain/document) by:
* Adding 'alias' tags in `api/` - for example `EthAccounts(context.Context) ([]string, error) //perm:read alias:eth_accounts`
* Allowing the 'alias' tag in https://github.com/filecoin-project/lotus/blob/master/gen/api/proxygen.go#L262
* Using reflect to go through fields of the generated [api proxy struct](https://github.com/filecoin-project/lotus/blob/master/api/proxy_gen.go#L104-L109) to do the aliasing here automatically

The few reasons to do that would be:
* Adding eth_ methods doesn't require going through multiple files; Aliases are in the same file as the API interface
* It's more self-documenting, and easier to add support to api-doc generation (api/docgen)
* Arguably easier to maintain (everything is in one place)

(this is just a suggestion, I'm fine with doing things the current way as well)

Reply:
Agree!

@ychiaoli18 For now I'd suggest we focus on buildling out the functionality, and later we can remove all the aliases from here and make them annotation-driven, as @magik6k proposes.

Reply:
Agree! I'll mark these as TODO so we can revisit after completing all the functionalities in the next milestones!